modules/nbde-tang-server-operator-identifying-url-web-console.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * security/nbde_tang_server_operator/nbde-tang-server-operator-identifying-url.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="identifying-url-nbde-tang-server-operator-using-web-console_{context}"]
+= Identifying URL of the NBDE Tang Server Operator using the web console
+
+You can identify URLs of Tang servers deployed with the NBDE Tang Server Operator from the OperatorHub using the {product-title} web console. After you identify the URLs, you use the `clevis luks bind` command on your clients containing LUKS-encrypted volumes you want to unlock automatically by keys advertised by the Tang servers. See the link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/security_hardening/configuring-automated-unlocking-of-encrypted-volumes-using-policy-based-decryption_security-hardening#configuring-manual-enrollment-of-volumes-using-clevis_configuring-automated-unlocking-of-encrypted-volumes-using-policy-based-decryption[Configuring manual enrollment of LUKS-encrypted volumes] section in the RHEL 9 Security hardening document for detailed steps describing the configuration of clients with Clevis.
+
+.Prerequisites
+
+* You must have `admin` privileges.
+* You deployed a Tang server using the NBDE Tang Server Operator on your OpenShift cluster.
+
+.Procedure
+
+. In the {product-title} web console, navigate to *Operators* -> *Installed Operators* -> *Tang Server*.
+
+. On the NBDE Tang Server Operator details page, select *Tang Server*.
++
+image::nbde-tang-server-operator-19-tangserver-details.png[NBDE Tang Server Operator details]
+
+. The list of Tang servers deployed and available for your cluster appears. Click on the name of the Tang server you want to bind with a Clevis client.
+
+. The web console displays an overview of the selected Tang server. You can find the URL of your Tang server in the `Tang Server External Url` section of the screen:
++
+image::nbde-tang-server-operator-21-tangserver-overview.png[NBDE Tang Server Operator overview of a Tang server]
++
+In this example, the URL of the Tang server is `http://34.28.173.205:7500`.
```

Comment:
See the doc guidelines on [Example URLs](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#example-urls).
```suggestion
. Use the value of the `Service External URL:` item without the `/adv` part. In this example, the URL of the Tang server is `\http://34.28.173.205:7500`.
```